share/python/driver.py

```diff
@@ -463,17 +463,16 @@ def energy(name, **kwargs):
     >>> molecule H2 {\n0 1\nH\nH 1 0.74\n}
     >>> set global basis cc-pVDZ
     >>> set global reference rohf
-    >>> energy('scf')
+    >>> scf_e, scf_wfn = energy('scf', return_wfn = True)
     >>> H2.set_multiplicity(3)
-    >>> psi4.MintsHelper().integrals()
-    >>> energy('detci', bypass_scf=True)
+    >>> psi4.MintsHelper(scf_wfn.basisset()).integrals()
```

Comment:
@bwb314, since an extra line is needed anyway for the MintsHelper, I think the more sensible example would be to use the extra line just do scf non-df. So fyi and why I'm changing it. Thanks for fixing this up.